### PR TITLE
Feature/split model

### PR DESCRIPTION
**Pull request by tavurth**
TODO:

- [x] Remove duplicate reward functions for alchemy and custom models
- [x] Remove duplicate reward weights for alchemy and custom models
- [x] Remove duplicate t2i and i2i models for alchemy and custom models
- [x] Add tests proving alchemy model gets 0 rewards on alchemy model step if it differs from any of the synapse variables
- [x] Test that a live mine running with the default configs gets a reward of 1.0 from the ModelDiveristyRewardModel
- [x] Test that a live miner running with the configs different to the default configs gets a reward of 0.0 from the ModelDiveristyRewardModel

### CODE REVIEW

File: neurons/validator/reward.py [hunk 1]
```diff
@@ -88,6 +84,66 @@ def apply_masking_functions(
     return rewards, event
 
 
+def process_manual_vote(
```
Comment (tavurth) [inline neurons/validator/reward.py]: Manual voting has been completely removed @KMFODA

File: neurons/validator/reward.py [hunk 2]
```diff
@@ -808,8 +891,8 @@ def load_models(self):
 
         # Set up mapping for the different synapse types
         self.mapping = {
-            "TEXT_TO_IMAGE": {"args": self.t2i_args, "model": self.t2i_model},
-            "IMAGE_TO_IMAGE": {"args": self.i2i_args, "model": self.i2i_model},
+            "text_to_image": {"args": self.t2i_args, "model": self.t2i_model},
```
Comment (tavurth) [inline neurons/validator/reward.py]: Should be uppercase

File: neurons/validator/validator.py [hunk 3]
```diff
@@ -196,23 +197,35 @@ def __init__(self):
...
+
         # Init reward function
-        self.reward_functions = [ImageRewardModel()]
+        self.reward_functions_alchemy = [
+            ImageRewardModel(),
+            ModelDiversityRewardModel(),
+        ]
+        self.reward_functions_custom = [
+            ImageRewardModel(),
+        ]
```
Comment (tavurth) [inline neurons/validator/validator.py]: This should be a single pipeline:

```py
self.reward_functions = [ImageRewardModel()]
```

Then:

```py
if self.model_type == ModelType.ALCHEMY:
  self.reward_functions.append(...)
```

Doing it this way (with two lists) leads to code duplication, and will be much harder to read @KMFODA

File: neurons/validator/validator.py [hunk 4]
```diff
@@ -196,23 +197,35 @@ def __init__(self):
...
             ],
             dtype=torch.float32,
         ).to(self.device)
 
-        self.reward_weights = self.reward_weights / self.reward_weights.sum(
-            dim=-1
-        ).unsqueeze(-1)
-
-        self.reward_names = ["image_reward_model"]
+        self.reward_weights_custom = torch.tensor(
```
Comment (tavurth) [inline neurons/validator/validator.py]: Same for the `reward_weights`, should be a single pipeline
